server/views/pages/recommendations/ppcs/determinateSentence/determinatePpudSentences.njk

```diff
@@ -12,17 +12,17 @@
 
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds-from-desktop">
-                <p class="govuk-body govuk-!-margin-bottom-8">
+                <p class="govuk-body">
                     {{ determinatePpudSentencesPageData.fullName }} has indeterminate and determinate sentences in PPUD. View the determinate sentences and then return to the indeterminate sentences to continue your booking.
                 </p>
                {% for group in determinatePpudSentencesPageData.determinateSentencesByCourt %}
-                    <h3 class="govuk-heading-m govuk-!-margin-bottom-2">Court case: {{ group.court }}</h3>
+                    <h3 class="govuk-heading-m govuk-!-margin-top-8">Court case: {{ group.court }}</h3>
 
                     {% for dateGroup in group.sentencesByDate %}
                         <p class="govuk-body govuk-!-margin-bottom-0">Date of sentence: {{ dateGroup.dateOfSentence }}</p>
 
                         {% for sentence in dateGroup.sentences %}
-                            <h4 class="govuk-heading-s">{{ sentence.offence.indexOffence }}</h4>
+                            <h4 class="govuk-heading-s govuk-!-margin-top-6 govuk-!-margin-bottom-3">{{ sentence.offence.indexOffence }}</h4>
```

Comment:
Isn't the default bottom margin for `govuk-heading-s` already 15px? I can't find the original definition, but I tried inspecting [the GOV.UK page on headings](https://design-system.service.gov.uk/styles/headings/) and it suggests as much.

Reply:
Answer: it is, but it's being overridden somewhere else in our CSS, so we need to re-override it to what it was originally. This is not the time to try to untangle why we do that, so happy to keep this as is.